packages/app/src/app/overmind/internalActions.ts

```diff
@@ -64,7 +64,7 @@ export const setPatronPrice: Action = ({ state }) => {
 
 export const setSignedInCookie: Action = ({ state }) => {
   document.cookie = 'signedIn=true; Path=/;';
-  identify('signed_in', true);
+  identify('signed_in', 'true');
```

Comment:
This one should be fine to have boolean now. We had it as boolean before, so for consistency I think we should have boolean again.
```suggestion
  identify('signed_in', true);
```